src/jit/decomposelongs.cpp

```diff
@@ -850,6 +850,90 @@ GenTree* DecomposeLongs::DecomposeArith(LIR::Use& use)
 }
 
 //------------------------------------------------------------------------
+// DecomposeArith: Decompose GT_ADD, GT_SUB, GT_OR, GT_XOR, GT_AND.
```

Comment:
This needs to be updated. Also, you should add a Note describing how the decomposition is done.